### ISSUE TITLE
No delete history in 'Web Browser' setting

### ISSUE THREAD

**Issue by not83** [ValveSoftware/SteamOS#472]
There is delete cookies, but not delete history in the 'Web browser' settings option.

If the delete cookies option is listed there, then I think it would be most logical to add this option too.

My system specs:
Steam Client built:
	Jan 14 2016, at 18:46:58
Steam versions
	1452827987
Steam API
	v017
SteamOS version
	SteamOS 2.0 update 1:2.60

CPU vendor
	GenuineIntel
CPU clock
	3.60GHz
System memory
	7.7GB
Video card
	NVIDIA Corporation GeForce GTX 660 Ti/PCIe/SSE2
Driver version
	4.5.0 NVIDIA 352.55
Video memory
	2.0 GB

**Comment by Plagman** [comment]
I don't believe the built-in browser even remembers history, which is why that option doesn't exist.

**Comment by not83** [comment]
There is a delete history option in the actual browser is there not?

When using the steam BPM browser when you are in the further most left tab
there is an option there.
Unless I'm imagining things

**Comment by Plagman** [comment]
You're right, this one might be specific to the BPM browser. I thought you were talking about the overlay browser for the regular client, but didn't realize this was a SteamOS bug entry and not steam-for-linux. It would indeed make sense to have a button to do that in that other settings dialog as well, will mark this as a feature request.

**Comment by not83** [comment]
😊
